### ISSUE TITLE
send-to-output doesn't resize a fullscreened window

### ISSUE THREAD

**Issue by dnkl** [ifreund/river#443]
Fullscreening a window, and then moving it to another output, doesn't resize the window.

Fullscreening the window:
```
[4226726.046] xdg_toplevel@25.configure(3440, 1440, array)
 dbg: wayland.c:632: xdg-toplevel: configure: size=3440x1440, states=[fullscreen, activated, tiled:left, tiled:right, tiled:top, tiled:bottom]
[4226726.128] xdg_surface@24.configure(1577)
 dbg: wayland.c:675: xdg-surface: configure
[4226726.168]  -> xdg_surface@24.ack_configure(1577)
[4226729.893]  -> xdg_surface@24.set_window_geometry(0, 0, 3440, 1440)
```
Moving the window to another output:
```
[4227835.375] wl_surface@3.leave(wl_output@16)
 dbg: wayland.c:559: unmapped from DP-1
[4227835.474] wl_surface@3.enter(wl_output@18)
 dbg: wayland.c:538: mapped on HDMI-A-1
```

After being moved, we see the window getting a configure event (reflecting the loss of keyboard focus). But, size isn't updated:
```
[4227871.287] wl_keyboard@20.leave(1583, wl_surface@3)
[4227871.309] xdg_toplevel@25.configure(3440, 1440, array)
 dbg: wayland.c:632: xdg-toplevel: configure: size=3440x1440, states=[fullscreen, tiled:left, tiled:right, tiled:top, tiled:bottom]
[4227871.337] xdg_surface@24.configure(1581)
 dbg: wayland.c:675: xdg-surface: configure
[4227871.359]  -> xdg_surface@24.ack_configure(1581)
```

**Comment by ifreund** [comment]
fixed by e6bb373240bc08668c8e6e14996a3f8765941158

**Comment by dnkl** [comment]
Verified. Thanks!
